src/Illuminate/Routing/Router.php

```diff
@@ -251,13 +251,29 @@ public function redirect($uri, $destination, $status = 301)
      * @param  array  $data
      * @return \Illuminate\Routing\Route
      */
-    public function view($uri, $view, $data = [])
+    public function view($uri, $view = null, $data = [])
```

Comment:
default parameters should always be last in order, how do you expect someone to call this if he wants the default value? 
Also this can be considered a breaking change

Reply:
... they are last?

Reply:
My bad 😁